server/oresat-live-software-server-service.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+/usr/sbin/startmonitor && /usr/bin/rx --dev mon0 /usr/local/sbin/oresat-live-software-server/public/video/ -e mp4 -v &
```

Comment:
It's probably better to have two systemd services:
1. start rx
2. start the web server (possibly this script)

I don't think `systemctl stop oresat-live-software-server` is going to kill the `rx` process that this script starts in the background.

---

I'm not sure if we even need this script. For the rx service, I believe we could have
```
ExecStartPre=/usr/sbin/startmonitor
ExecStart=/usr/bin/rx --dev mon0 /usr/local/sbin/oresat-live-software-server/public/video/ -e mp4 -v
```

And for the server service (double-check the paths of npm and node, I just assumed `/usr/bin/`):
```
[Unit]
...
Requires=oresat-rx.service
After=oresat-rx.service

[Service]
WorkingDirectory=<path_to_server_directory>
ExecStartPre=/usr/bin/npm i
ExecStartPre=mkdir -p public/video
ExecStart=/usr/bin/node index.js video/
```

Reply:
I mean I will test it but seeing as it kills the parent shell process the child should also die.
Otherwise, don't mind having two services, makes sense, same with these systemd things you recommend, can implement those

As far as needing the script, in my tests I needed it. I can't give you good explanation for why though but transmit didn't work reliably until i ran it, so

Reply:
> needing the script

I mean as opposed to doing it all in the systemd service unit files, since it's pretty short and the script just consists of a couple of prep commands

Reply:
ugh, it might be a good standalone utility for anyone wanting to test the shit manually without having to copy/paste from systemd files. personally, ive found it useful in testing. and "a few" is like 5 lines of  command invocations, so it kinda feels like it deserves its own script, even if stripped down from the unneeded extra args and cleanup shit

Reply:
Makes sense to me.